### PR TITLE
Add ImageClassifier example with p5 preload

### PR DESCRIPTION
**Pull request by NHibiki**
This is the pull request related to https://github.com/ml5js/ml5-library/pull/288. 

- Adding the example for Image Classification with p5 preload.

### CODE REVIEW

File: p5js/ImageClassification/ImageClassification_Preload/sketch.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+let classifier;
+
+// A variable to hold the image we want to classify
+let img;
+
+function preload() {
+  // Initialize the Image Classifier method with MobileNet.
+  classifier = ml5.imageClassifier('mobilenet');
+  // Set Status after the model is loaded
+  select('#status').html('Model Loaded');
```
Comment (shiffman) [inline p5js/ImageClassification/ImageClassification_Preload/sketch.js]: We can remove this from the example entirely (the DOM element included) as the model can be assumed to be loaded!

File: p5js/ImageClassification/ImageClassification_Preload/sketch.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+// A variable to hold the Image Classifier
+let classifier;
+
+// A variable to hold the image we want to classify
+let img;
+
+function preload() {
+  // Initialize the Image Classifier method with MobileNet.
+  classifier = ml5.imageClassifier('mobilenet');
```
Comment (shiffman) [inline p5js/ImageClassification/ImageClassification_Preload/sketch.js]:
```suggestion
  classifier = ml5.imageClassifier('MobileNet');
```

Small thing but this is the preferred capitalization I believe.

File: p5js/ImageClassification/ImageClassification_Preload/sketch.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  // Set Status after the model is loaded
+  select('#status').html('Model Loaded');
+  // Load the image
+  img = loadImage('bird.jpg');
+}
+
+function setup() {
+  noCanvas();
```
Comment (shiffman) [inline p5js/ImageClassification/ImageClassification_Preload/sketch.js]: There should be a canvas and we should draw the image on it!

File: p5js/ImageClassification/ImageClassification_Preload/sketch.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+// A function to run when we get any errors and the results
+function gotResult(err, results) {
+  // Display error in the console
+  if (err) {
+    console.error(err);
+  }
+  // The results are in an array ordered by confidence.
+  select('#result').html(results[0].label);
```
Comment (shiffman) [inline p5js/ImageClassification/ImageClassification_Preload/sketch.js]: Let's simplify and remove the HTML elements from `index.html` and just use p5's `createP()` or `createDiv()` or. . .

**Comment by NHibiki** [comment]
@shiffman Thanks for giving advice in details 👍. I think the modified version shows the great power of p5. Do you think it is necessary to change other examples? (Since I borrowed those codes from `p5js/ImageClassification/ImageClassification`)

**Comment by NHibiki** [comment]
@shiffman any update on this pull request? ➿

File: p5js/ImageClassification/ImageClassification_Preload/sketch.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+function preload() {
+  // Initialize the Image Classifier method with MobileNet.
+  classifier = ml5.imageClassifier('MobileNet');
+  // Load the image
+  img = loadImage('bird.jpg');
+}
+
+function setup() {
+  // Display the image
```
Comment (shiffman) [inline p5js/ImageClassification/ImageClassification_Preload/sketch.js]: I think this is missing a `createCanvas()` here 😄

**Comment by shiffman** [comment]
Hi @NHibiki apologies for the delay. We had a discussion about how to merge new examples in when the new features of the library are not yet published to NPM. Amusingly I can't seem to remember where we ended up with this, perhaps @joeyklee's brain works better than mine 😆.

This is ready to go, however, I think that we should move towards using `preload()` as the default for p5 examples. The callback methodology can be in the documentation and any plan JS examples.

I think you should go ahead and add `preload()` support for all other models then we can do a big pass over all the examples when we are ready to do a release. How does that sound?

**Comment by joeyklee** [comment]
@NHibiki - thanks for adding this example! @shiffman I've merged in these changes for now. Let's add this to the list of things to remember that we should:
1. `add preload() support` for all the other models 
2. do a big pass over all the examples and restructure them to use preload() for the p5js examples

Thanks!

**Pull request merged**